include/debug_scani2c.h

```diff
@@ -20,24 +21,17 @@ void scanI2C() {
         error = Wire.endTransmission();
 
         if (error == 0) {
-            Serial.print("I2C device found at address 0x");
-            if (address < 16) Serial.print("0");
-            Serial.print(address, HEX);
-            Serial.println("  !");
+            OSW_LOG_I("I2C device found at address 0x", address < 16 ? '0' : '', msg, String(address, HEX), " !");
 
             nDevices++;
         } else if (error == 4) {
-            Serial.print("Unknown error at address 0x");
-            if (address < 16) Serial.print("0");
-            Serial.println(address, HEX);
+            OSW_LOG_E("Unknown error at address 0x", address < 16 ? "0" : "", String(address, HEX));
         }
     }
     if (nDevices == 0)
-        Serial.println("No I2C devices found\n");
+        OSW_LOG_I("No I2C devices found");
     else
-        Serial.println("done\n");
-
-    delay(5000);  // wait 5 seconds for
```

Comment:
Where did this go?

Reply:
As far as I can tell this sleep has no purpose today anymore and just slows down rapid scanning. So I removed it.